aiplatform/src/test/java/aiplatform/CreatePipelineJobEmbeddingModelTuningSampleTest.java

```diff
@@ -0,0 +1,149 @@
+  private static final int ITERATIONS = 300;
+  private String pipelineJobName;
+  private ByteArrayOutputStream bout;
+  private PrintStream originalPrintStream;
+
+  private static void requireEnvVar(String varName) {
+    String errorMessage =
+        String.format("Environment variable '%s' is required to perform these tests.", varName);
+    assertNotNull(errorMessage, System.getenv(varName));
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
+    requireEnvVar("UCAIP_PROJECT_ID");
+  }
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+    originalPrintStream = System.out;
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown()
+      throws IOException, InterruptedException, TimeoutException, ExecutionException {
+    PipelineServiceSettings pipelineServiceSettings =
+        PipelineServiceSettings.newBuilder().setEndpoint(API_ENDPOINT).build();
+
+    try (PipelineServiceClient pipelineServiceClient =
+        PipelineServiceClient.create(pipelineServiceSettings)) {
+      // Cancel the PipelineJob
+      pipelineServiceClient.cancelPipelineJob(pipelineJobName);
+      TimeUnit.MINUTES.sleep(2);
+
+      // Delete the PipelineJob
+      int retryCount = 3;
+      while (retryCount > 0) {
+        retryCount--;
+        try {
+          OperationFuture<Empty, DeleteOperationMetadata> operationFuture =
+              pipelineServiceClient.deletePipelineJobAsync(pipelineJobName);
+          operationFuture.get(300, TimeUnit.SECONDS);
+
+          // if delete operation is successful, break out of the loop and continue
+          break;
+        } catch (StatusRuntimeException e) {
+          // wait for another 1 minute, then retry
+          System.out.println("Retrying (due to unfinished cancellation operation)...");
+          TimeUnit.MINUTES.sleep(1);
+        } catch (Exception otherExceptions) {
+          // other exception, let them throw
+          throw otherExceptions;
+        }
+      }
+    }
+
+    System.out.flush();
+    System.setOut(originalPrintStream);
+  }
+
+  @Test
+  public void createPipelineJobEmbeddingModelTuningSample() throws IOException {
+    final String pipelineJobDisplayName =
+        String.format(
+            "temp_create_pipeline_job_test_%s",
+            UUID.randomUUID().toString().replaceAll("-", "_").substring(0, 26));
+    CreatePipelineJobEmbeddingModelTuningSample.createEmbeddingModelTuningPipelineJob(
+        API_ENDPOINT,
+        PROJECT,
+        BASE_MODEL_VERSION_ID,
+        TASK_TYPE,
+        pipelineJobDisplayName,
+        OUTPUT_DIR,
+        QUERIES,
+        CORPUS,
+        TRAIN_LABEL,
+        TEST_LABEL,
+        BATCH_SIZE,
+        ITERATIONS);
+
+    String got = bout.toString();
+    System.out.println("hylee:: " + got);
+    assertThat(got).contains(pipelineJobDisplayName);
```

Comment:
Using STDOUT for test assertions is not recommended. Consider a case when the tested code changes printing string in a way that the asserted substring remains but totally changes the logic.
Please, refactor `createEmbeddingModelTuningPipelineJob()` to return status code that can be validated.